python/cuml/cuml/dask/ensemble/randomforestregressor.py

```diff
@@ -364,19 +364,15 @@ def partial_inference(self, X, delayed, **kwargs):
         partial_infs = self._partial_inference(
             X=X, op_type="regression", delayed=delayed, **kwargs
         )
-
-        def reduce(partial_infs, workers_weights, unique_classes=None):
-            regressions = dask.array.average(
-                partial_infs, axis=1, weights=workers_weights
-            )
-            merged_regressions = regressions.compute()
-            return merged_regressions
-
-        datatype = (
```

Comment:
`datatype` is unused in `self.apply_reduction`

Reply:
@viclafargue could you maybe comment on that?